src/options.ts

```diff
@@ -18,6 +18,12 @@ export interface Options {
    */
   jsii?: JsiiOutputOptions;
 
+  /**
+   * Package name
+   * @default "generated"
```

Comment:
Given the issue this creates with multiple packages, any way we can provide some other default?

We can do a random string (not ideal but will be fine) or we could do some hash of the source (maybe better)

Reply:
Doing a hash of the entrypoint now:

```
crypto.createHash('sha256').update(basepath, 'utf8').digest('hex')
```